Dockerfile

```diff
@@ -14,6 +14,9 @@ RUN apk update \
         && apk del openjdk11 \
         && rm -rf /var/cache/apk/*
 
+# delete bench utils for leaner container
+RUN rm -rf /lrsql/bench*
```

Comment:
That simple?

Reply:
yeah well there's two things. `bench.jar` and `bench/*.json`. This seems to remove both.